### PR TITLE
MONDO:0000424 inborn vitamin B12 deficiency [Merge]

### LINKED ISSUE #7353: MONDO:0000424 inborn vitamin B12 deficiency [Merge]
**Mondo term (ID and Label)**
MONDO:0000424  inborn vitamin B12 deficiency

**Reason for deprecation**
The DOID term referenced for this Mondo ID is "vitamin B12 deficiency" which is a child of 'inherited metabolic disorder' and uses a syn of "cobalamin deficiency" 

**Term to be merged with**
MONDO:0019220 inborn disorder of cobalamin metabolism and transport  (Again, based on the DOID hierarchy and syns, this seems to be a conceptual match, http://purl.obolibrary.org/obo/DOID_0050731 ) And it appears there is some overlap in children between this Mondo:0000424

Or 

MONDO:0020696  vitamin B12 deficiency (the NCI definition on MONDO:0000424 is the same code, C131684, as the exact match to this concept, but this B12 deficiency is stated that it can be acquired or inherited, so i am not convinced this is the right merge.)

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

I'm not sure if vitamin B12 really is exactly equivalent with cobalamin... the NCI definition for cobalamin starts with : "An essential nutrient and natural water-soluble vitamin of the B-complex family that must combine with an intrinsic factor for absorption by the intestine, Vitamin B12 (cyanocobalamin) is necessary for hematopoiesis..." (thus B12  == _cyano_cobalamin??) but then B12 is used as a synonym by NCI for cobalamin. https://ncithesaurus.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C939

### PR DESCRIPTION
**Pull request by katiermullen**
closes #7353

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -435350,21 +435336,35 @@ is_a: MONDO:0019250 {source="Orphanet:79169"} ! inborn disorder of biogenic amin
 [Term]
 id: MONDO:0019220
 name: inborn disorder of cobalamin metabolism and transport
+def: "An inherited metabolic disease affecting cobalamin (vitamin B12) intestinal absorption, transport in the blood, uptake by peripheral cells or cellular metabolism." [PMID:35337626]
 subset: clingen {source="MONDO:CLINGEN"}
 subset: disease_grouping
 subset: gard_rare {source="GARD:18951", source="MONDO:GARD"}
+subset: gard_rare {source="MONDO:GARD"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:435343]: this line is duplicated

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -435350,21 +435336,35 @@ is_a: MONDO:0019250 {source="Orphanet:79169"} ! inborn disorder of biogenic amin
...
 xref: Orphanet:79171 {source="MONDO:equivalentTo"}
 xref: UMLS:C5681844 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1826150"}
+is_a: MONDO:0005528 {source="DOID:0050731", source="HP:0100502", source="MONDO:Entailed", source="MONDO:Redundant"} ! inborn vitamin metabolic disorder
 is_a: MONDO:0017758 {source="Orphanet:79171"} ! disorder of vitamin and non-protein cofactor absorption and transport
+intersection_of: MONDO:0019052 ! inborn errors of metabolism
+intersection_of: disease_has_basis_in_disruption_of GO:0009235 ! cobalamin metabolic process
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0019220 {source="MONDO:CLINGEN"}
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7353" xsd:anyURI
+property_value: IAO:0000589 "inborn vitamin B12 deficiency (disease)" xsd:string
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:435367]: Do you know where this came from?

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@katiermullen 
Very minor feedback but I am sending it back to you so it is on your radar (since you haven't done a lot of term merging in Mondo).

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:435367]: I think it is given when the label is the same as another label in obo (e.g. as a phenotype). 
I think we can remove this annotation and make sure that the name is a synonym

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:435343]: removed duplicated line.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:435367]: removed this annotation and made sure that the name "inborn vitamin B12 deficiency (disease)" is listed as a synonym.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:435343]: I forgot to tell you (sorry! I got distracted). 
The easy way to deal with things like this is by running a normalization step

**Review by sabrinatoro — APPROVED** [review]
Perfect @katiermullen ! 
Approving and merging.

**Pull request merged**